app/javascript/articles/components/__tests__/SaveButton.test.jsx

```diff
@@ -3,17 +3,31 @@ import { fireEvent, render } from '@testing-library/preact';
 import { axe } from 'jest-axe';
 import { SaveButton } from '../SaveButton';
 
+it('should not show button when post owned by current user', () => {
+  const article = { class_name: 'Article', id: 1 };
+  const { queryByText } = render(
+    <SaveButton article={article} saveable={false} />,
+  );
+  const saveButton = queryByText('Saved');
+
+  expect(saveButton).toBeNull();
+});
+
 it('should have no a11y violations', async () => {
   const article = { class_name: 'Article', id: 1 };
-  const { container } = render(<SaveButton article={article} />);
+  const { container } = render(
+    <SaveButton article={article} saveable={true} />,
```

Comment:
Without `saveable={true}` on this and subsequent tests, they all fail.

Reply:
If we wanted to avoid always having to pass `saveable` prop we could just give it a default value in `SaveButton`, but I think it's fine either way 🤷‍♀️